src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs

```diff
@@ -33,17 +38,40 @@ public void ConfigureServices(IServiceCollection services)
             Log.Logger.Information("Api Resources: {@resources}", resources);
             Log.Logger.Information("Identity Resources: {@identityResources}", identityResources);
 
+            // configures IIS out-of-proc settings (see https://github.com/aspnet/AspNetCore/issues/14882)
```

Comment:
Don't think we need the IISOptions here as we will not be using that proxy. This will also simply the setup code.